### ISSUE TITLE
Examples Reproducibility

### ISSUE THREAD

**Issue by chaosicong** [MaginnGroup/PyLAT#29]
Hi,Mike! I have a question want to require you. If I want to repeat your outcome,whether should I modify the Examples'  .in and .data file.Can I just put the .in and .data files into lammps and run them?I'm new to lammps simulation, and I've read your literature, which mentions a lot of ensemble use, but there are a few fixed ensemble commands in your example that have been #dropped, do I need to modify it?

**Comment by mike5603** [comment]
In general with molecular dynamics you will want to equilibrate your system using npt. In this case, the mol.data files contain a system that has already been equilibrated so you should be able to run the examples as is. The only different example is the viscosity, which requires multiple runs to calculate in an effective manner so you will have to use the input generator script

**Comment by chaosicong** [comment]
Thank you for your guidance! I have reviewed your literature on Pylat and provided detailed simulation information on the dielectric constant of acetonitrile in the supplementary information. It is mentioned that the first simulation was conducted at 100ps under the 298K, NPT ensemble; Afterwards, the NVT ensemble was used and ran for 20 ns to obtain the dielectric constant; In the example you provided, the. in file for the dielectric constant of acetonitrile was taken after balancing. If I want to follow the literature from start to finish, I should make modifications to the. in file, right? Looking forward to your reply, thank you again for your guidance!

**Comment by mike5603** [comment]
Yes. In general I would run the different equilibration steps as separate simulations so I could monitor the output of those steps / if I am calculating different properties with different requirements, I could equilibrate once and use the outputs in multiple simulations

**Comment by chaosicong** [comment]
I am particularly excited to receive your reply.For me, I only have half a year left in my research career, and working in this new field of molecular dynamics is a bit difficult for me, and if you can share with me all the simulation files you have for this paper, it will definitely be able to learn a lot  quickly. I will be grateful for that! Looking forward to hearing from you

**Comment by mike5603** [comment]
I am not in the field any longer so I don't have ready access to my files, but there are some tutorials for lammps you can follow here https://www.lammps.org/tutorials.html

**Comment by chaosicong** [comment]
Sorry to bother you again, but after reading the Pylat article again, I ran into a problem trying to repeat it, and in your support message there was the following sentence, "The acetonitrile system was modeled using the general Amber force field1 for intramolecular terms as well as non-bonded Lennard-Jones parameters. "I tried to use the "import gaff.lt" in moltemplate, but I can't get the same force field parameters as the case you provided, is there something wrong with me, I have tried for a long time and have not solved it, I beg you to guide me. I'm almost unable to graduate, hahaha
